README.md

```diff
@@ -27,21 +27,33 @@ fn main() {
     let nbt = nbt!("root nbt_inner name", {
         "float": 1.0,
         "key": "value",
-        "long_array": [L; 1],
-        "int_array": [Int; 1],
+        "long_array": [L; 1, 2],
+        "int_array": [Int; 1, 10, 25],
+        "byte_array": [B; 0, 1, 0, 0, 1],
         "list": ["a", "b", "c"],
         "nbt_inner": {
             "key": "sub value"
         }
     });
 
+    // The macro can also take variables to use data calculated at runtime
+    let var_as_key = "some_key".to_owned();
+    let other_key = "other_key".to_owned();
+    let value = vec![0, 1, 2];
+    let other_value = vec![3, 2, 1];
+    let nbt =  nbt!("root", {
+        var_as_key: "wohoo!",
+        "the_other_way": value,
+        other_key: other_value,
+    });
```

Comment:
IMO, that is logical. We don't need to take space in readme to show such thing.

Reply:
How? You are used to this kind of macro already from `serde_json` `json!` macro, but if someone never used `serde_json` and is new to this type of things he might think it's only for literals and not know variables can be used too.